src/usb/usb_processing.c

```diff
@@ -39,7 +39,7 @@ extern struct timer_descriptor TIMER_0;
  * Amount of time to wait before an outstanding operation times out
  * (if the client is closed).
  */
-#define USB_OUTSTANDING_OP_TIMEOUT_MS (7000)
+#define USB_OUTSTANDING_OP_TIMEOUT_MS (300)
```

Comment:
The previous value was 500. Why lower it? Seems a little risky, a sit leaves less time for the host to send the poll/retry msg.

Reply:
ah, I must have misremembered.

Reply:
The commit msg also mentions the host polls every 300ms, but it's every 200ms.

Reply:
It mentions that the firmware expects within 300ms, after updating to 500ms I woudl say it expects a keepalive within 500ms. Of course the app can send every 200ms if it wants to.

Reply:
anyway, comment addressed